### PR TITLE
Update 6.8-beta link to 6.8 in reqs for installation in ipv6 network …

### PR DESCRIPTION
**Pull request by suyogsainkar**
…file

CC: @spetrosi for visibility

### CODE REVIEW

**Comment by suyogsainkar** [comment]
Quick update to the 6.8-beta link; merging.

File: guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc [hunk 1]
```diff
@@ -19,4 +19,4 @@ ifeval::["{build}" != "satellite"]
 Note that this requirement is for Katello users only.
 endif::[]
 
-* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8-beta/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
+* You must configure {Project} to use this IPv4 HTTP proxy server as the default proxy. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html/installing_satellite_server_from_a_connected_network/performing-additional-configuration#adding-a-default-http-proxy_satellite[Adding a Default HTTP Proxy to Satellite].
```
Comment (spetrosi) [inline guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:22]: This should be a {ProductVersion} attribute, not a plain text version.

Comment (spetrosi) [reply guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:22]: I also noticed that the PR is opened for SATELLITE-6.8 branch. We definitely need to make this change to master to not return to it and use the {ProductVersion} attribute.

Comment (suyogsainkar) [reply guides/common/modules/con_requirements-for-installation-in-an-ipv6-network.adoc:22]: Sorry, fixed in https://github.com/theforeman/foreman-documentation/pull/302 and will backport to 6.8.

**Pull request merged**